src/main/scala/com/simple/jdub/Transaction.scala

```diff
@@ -26,6 +26,38 @@ class Transaction(connection: Connection) extends Queryable {
     onRollback.foreach(_())
   }
 
+  /**
+   * Roll back the transaction to a savepoint.
+   */
+  def rollback(savepoint: Savepoint) {
```

Comment:
Procedure syntax is deprecated. Please change this and the below function to annotate the `Unit` return type.